### DISCUSSION TITLE
Masking out background for SSIM evaluation

### DISCUSSION THREAD

**Discussion by briannghiem** [facebookresearch/fastMRI#323] [Q&A] [answered]
Hello,
Could you please share the code for the post-hoc background-masking algorithm that was described in the 2020 fastMRI paper (Section III.B - Quantitative Results)? I'm looking into ways of handling background noise when evaluating image quality metrics and wanted to know how your group approached this.

Best,

**Comment by mmuckley** [comment, marked as answer]
Hello @briannghiem, this workstream was led by Bruno Riemenschneider, who was a postdoc at the time and has long since left. As stated in the [supplemental material](https://ieeexplore.ieee.org/ielx7/42/9526230/9420272/supp1-3075856.pdf?arnumber=9420272), first the mask was calculated, and then all images in the challenge set were manually inspected to verify that the masking did not remove large anatomical regions.

As far as I can tell, the code was not shared on our (Meta) side and so I am unable to reproduce the masking algorithm.

**Reply by briannghiem** [reply]
Thank you for letting me know!
